tests/unit/onyxTest.js

```diff
@@ -494,15 +494,15 @@ describe('Onyx', () => {
         // Given the invalid data object with onyxMethod='multiSet'
         const data = [
             {onyxMethod: 'set', key: ONYX_KEYS.TEST_KEY, value: 'four'},
-            {onyxMethod: 'multiSet', key: ONYX_KEYS.OTHER_TEST, value: {test2: 'test2'}},
+            {onyxMethod: 'murge', key: ONYX_KEYS.OTHER_TEST, value: {test2: 'test2'}},
         ];
 
         try {
             // When we pass it to Onyx.update
             Onyx.update(data);
         } catch (error) {
             // Then we should expect the error message below
-            expect(error.message).toEqual('Invalid onyxMethod multiSet in Onyx update.');
+            expect(error.message).toEqual('Invalid onyxMethod murge in Onyx update.');
```

Comment:
murge?! 😂

Reply:
Haha - yeah open to ideas for better names for a "wrong" onyxMethod.